docs/modules/usage/llms/litellm-proxy.md

```diff
@@ -0,0 +1,55 @@
+
```

Comment:
Should this show up under Providers? 
We need to add it to the sidebar.ts for it to show up on the doc site.

Reply:
If it's going under "Providers", should it be called "LiteLLM" or "LiteLLM Proxy?

Reply:
Let's add it to side bar and called it "LiteLLM Proxy" under provider?

Reply:
Sounds good.